website/sidebars.js

```diff
@@ -799,6 +797,22 @@ const sidebarSettings = {
             "guides/best-practices/how-we-structure/5-the-rest-of-the-project",
           ],
         },
+        {
+          type: "category",
+          label: "How we style our dbt projects",
+          link: {
+            type: "doc",
+            id: "guides/best-practices/how-we-style/0-how-we-style-our-dbt-projects",
```

Comment:
I have a strong distaste for secret documents hiding under the header (if you click the down arrow to expand it, you'll never know there is another page there), but I don't think this is the place to fight that fight

Reply:
yea let's sidebar that for another time, as this is in place on a bunch of stuff in the devhub already and i (v) soft disagree but am open to being convinced in the other direction, i don't think i hold an opinion v strongly here